src/services/open_ai/query_openai.py

```diff
@@ -387,14 +397,20 @@ def invoke_tool(ctx:Context, tool, input):
         answer = serper.run(input)
         ctx.log(f'SERPER search result: {answer}')
 
-        return answer
+        return answer, False
 
     if tool_canon.startswith('WEATHER'):
         answer = invoke_weather_search(ctx, input)
 
-        return answer
+        return answer, False
+    
+    if tool_canon.startswith('ALERT'):
+        ctx.set_stat('tools-flow:tool-alert', 1)
+        invoke_alert_tool(ctx, input, message)
+        return "succesfully added the timer", True
```

Comment:
Message: alert added successfully.

Best if you include the actual time it is set to.

Reply:
Relative time? I don't think it's needed, if we had an absolute time, it would be cool